packages/flutter/test/widgets/page_route_builder_test.dart

```diff
@@ -91,13 +93,23 @@ class ModalPage extends StatelessWidget {
 }
 
 void main() {
+  testWidgets('Material2 - Barriers show when using PageRouteBuilder', (WidgetTester tester) async {
+    await tester.pumpWidget(const TestPage(useMaterial3: false));
+    await tester.tap(find.byType(FloatingActionButton));
+    await tester.pumpAndSettle();
+    await expectLater(
+      find.byType(TestPage),
+      matchesGoldenFile('m2_page_route_builder.barrier.png'),
+    );
+  });
+
   testWidgets('Barriers show when using PageRouteBuilder', (WidgetTester tester) async {
```

Comment:
nit: Should we here also show 'Material3 - ' at the beginning?

Reply:
Oops, yes this one should be labeled Material3.